ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -520,6 +510,7 @@ TFLiteCore::TFLiteCore (const char * _model_path, const char * accelerators)
 
   setAccelerator (accelerators);
 
+  g_message ("nnapi = %d, accl = %s", use_nnapi, get_accl_hw_str(accelerator));
```

Comment:
this line is duplicated with below line (DBG)

Reply:
Updated.